### ISSUE TITLE
Installing DebugBar for Laravel 5.8.31 on WSL 18.04 => slow response

### ISSUE THREAD

**Issue by FamilioDev** [barryvdh/laravel-debugbar#950]
Installing DebugBar on fresh laravel installation (Laravel 5.8, Nginx + PHP7.2 FPM, WSL Ubuntu 18.04, Windows 1903 18362.239) makes laravel app unusable due to slow response (more than minute).
Uninstalling DebugBar makes laravel fast again.
PHP slow log:
```
[12-Aug-2019 23:46:50.741704]  [pool tim] pid 1790 script_filename = /home/tim/web/test/public/index.php
[0x00007f6de361c140] fastcgi_finish_request() /home/tim/web/test/vendor/symfony/http-foundation/Response.php:379
[0x00007f6de361c0e0] send() /home/tim/web/test/public/index.php:58
```

**Comment by barryvdh** [comment]
Are you logging a lot of data? It writes the data to file, so if file access is very slow, that might occur.

**Comment by FamilioDev** [comment]
> 
> 
> Are you logging a lot of data? It writes the data to file, so if file access is very slow, that might occur.

No. This is fresh installation without any additional logic or tuning. I enabled debug logging afterwards to trace problem.
By the I have an update. I tried this on Windows 1809 and it worked without any problem. So this seems to be 1903 problem. I'll try a fresh 1903 later to see if it's generally 1903 problem or is it my specific config causing it.
